packages/mobile/src/geth/saga.test.ts

```diff
@@ -0,0 +1,41 @@
+import { GenesisBlockUtils, StaticNodeUtils } from '@celo/contractkit'
+import GethBridge from 'react-native-geth'
+import { expectSaga } from 'redux-saga-test-plan'
+import { InitializationState } from 'src/geth/reducer'
+import { _waitForGethInit, GethInitOutcomes, initGethSaga } from 'src/geth/saga'
+import { createMockStore } from 'test/utils'
+
+describe(initGethSaga, () => {
+  beforeAll(() => {
+    jest.useRealTimers()
+  })
+
+  beforeEach(() => {
+    const getStaticNodesAsync = StaticNodeUtils.getStaticNodesAsync as jest.Mock
+    getStaticNodesAsync.mockReturnValue(Promise.resolve('enodes'))
+    const getGenesisBlockAsync = GenesisBlockUtils.getGenesisBlockAsync as jest.Mock
+    getGenesisBlockAsync.mockReturnValue(Promise.resolve({}))
+    const MockGethBridge = (GethBridge as unknown) as Record<string, jest.Mock>
+    MockGethBridge.startNode.mockClear()
+  })
+
+  it('initializes the bridge and starts the node', async () => {
+    const state = createMockStore({}).getState()
+    await expectSaga(_waitForGethInit)
+      .withState(state)
+      .returns(GethInitOutcomes.SUCCESS)
+      .run()
+    expect(state.geth.initialized).toEqual(InitializationState.INITIALIZED)
```

Comment:
Checking for the saga to `put` certain actions is more inline with how expectSaga is meant to be used but if this works then it's fine too :)